src/core/src/sim/mc/moves/mod.rs

```diff
@@ -27,20 +28,28 @@ pub trait MCMove {
     fn prepare(&mut self, system: &mut System, rng: &mut Box<Rng>) -> bool;
 
     /// Get the cost of performing this move on `system`. For example in
-    /// simple NVT simulations, this cost is the energetic difference over
-    /// `beta`. The cost must be dimmensionless, and will be placed in an
-    /// exponential. The `cache` should be used to compute the cost, or the
+    /// simple NVT simulations, this cost is the energetic difference between 
+    /// the new and the old state times beta. The cost must be dimmensionless. 
+    ///
+    /// Note that the cost will be placed in an exponential with a negative sign.
+    /// For NVT using the Metropolis criterion: 
+    /// cost = beta*(U_new - U_old) -> P_acc = min[1, exp(-cost)]. 
+    ///
+    /// The `cache` should be used to compute the cost, or the
     /// `cache.unused` function should be used to ensure that the cache is
-    /// updated  as needed after this move.
+    /// updated as needed after this move.
     fn cost(&self, system: &System, beta: f64, cache: &mut EnergyCache) -> f64;
 
-    /// Effectivelly apply the move, if it has not already been done in
+    /// Effectivelly apply the move if it has not already been done in
     /// `prepare`.
     fn apply(&mut self, system: &mut System);
 
-    /// Restore the system to it's initial state, if it has been changed in
+    /// Restore the system to it's initial state if it has been changed in
     /// `prepare`.
     fn restore(&mut self, system: &mut System);
+    
+    /// Update the displacement
+    fn change_range(&mut self, scaling_factor: Option<f64>);
```

Comment:
This could have a better name, how about something like `update_amplitude(&mut self, factor: Option<f64>)`. 

Also, what does the Option means here? If it is `None`, no changes should be performed?